src/components/Customer.js

```diff
@@ -33,85 +43,153 @@ export class Customer extends Component {
   componentDidUpdate() {
     this.refreshList();
   }
+
   render() {
     const { cust, cusid, cusname, cusage, cusaddress } = this.state;
     let addModalClose = () => this.setState({ addModalShow: false });
```

Comment:
I assume these functions do not change so you should use 'const' instead of 'let'